medical_lab/models/medical_lab_test_criterion.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright 2012-2013 Federico Manuel Echeverri Choux
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class MedicalLabTestCriterion(models.Model):
+
```

Comment:
PEP 8 actually says to put a line here. As Odoo devs, we disobey this rule a lot. My line in the sand is when it is more just a name, description, or inherit.